### ISSUE TITLE
Rtm.start is deprecated

### ISSUE THREAD

**Issue by bsoule** [beeminder/beebot#29]
and going away, and there are tons of warnings printed in the logs because of it, and maybe it's also breaking everything because messaging doesn't seem to actually work now.

**Comment by bsoule** [comment]
ok, #32, #33, #34, #35, #36 were all incremental work toward this.

We're now connecting successfully to the heroku redis instance, and we're also able to send zeno DMs and channel messages again. 

This is good enough for now.

I updated those methods to use `chat.postMessage` for both of those. It worked to use node's slack-sdk `rtm.sendMessage` api in one case, but not the other (for no clear reason.. sigh), so I changed them both to using the newer `chat.postMessage`, because I think that's more in line with what Slack wants us to do anyway. 

I did not go through and update other instances of `sendMessage` or other ways of sending messages in the slackbot app, because the best I can tell, I think most of the other bits work for the most part (i.e. tocks, and bids, and rolls, oh my).
